### PR TITLE
Configurable excel tmp dir

### PR DESCRIPTION
**Pull request by loostro**

### CODE REVIEW

File: Resources/templates/CommonAdmin/ExcelAction/ExcelBuilderAction.php.twig [hunk 1]
```diff
@@ -45,7 +45,7 @@ class ExcelController extends ListController
         $writer = $phpexcel->createWriter($phpExcelObject, '{{ builder.filetype }}');
         $response = new StreamedResponse(
             function () use ($writer) {
-                $tempFile = $this->get('kernel')->getRootDir().'/cache/' . 
+                $tempFile = '{{ builder.generator.bundleConfig.excel_tmp_dir }}/' .
```
Comment (matejvelikonja) [inline Resources/templates/CommonAdmin/ExcelAction/ExcelBuilderAction.php.twig]: is there a reason you didn't just use `$this->get('kernel')->getCacheDir()`?

Comment (loostro) [inline Resources/templates/CommonAdmin/ExcelAction/ExcelBuilderAction.php.twig]: @matejvelikonja no, no reason. actually that would be much simpler, I haven't thought of that :) I'll fix it :) thanks

Comment (matejvelikonja) [inline Resources/templates/CommonAdmin/ExcelAction/ExcelBuilderAction.php.twig]: np :)

Comment (loostro) [inline Resources/templates/CommonAdmin/ExcelAction/ExcelBuilderAction.php.twig]: @matejvelikonja this is what happens when you code till 2AM :P

**Pull request merged**